apps/opik-backend/src/main/java/com/comet/opik/api/filter/ExperimentsComparisonValidKnownField.java

```diff
@@ -12,6 +12,7 @@ public enum ExperimentsComparisonValidKnownField implements Field {
 
     OUTPUT(OUTPUT_QUERY_PARAM, FieldType.STRING),
     FEEDBACK_SCORES(FEEDBACK_SCORES_QUERY_PARAM, FieldType.FEEDBACK_SCORES_NUMBER),
+    FEEDBACK_SCORES_EMPTY(FEEDBACK_SCORES_EMPTY_QUERY_PARAM, FieldType.EMPTY),
```

Comment:
What about having it as `FEEDBACK_SCORES_LIST` and `FieldType.LIST`?

Reply:
I gave some more thought, changed the logic a bit, the parameter is now `FEEDBACK_SCORE_COUNT` of type `FieldType.NUMBER`